test/e2e/__init__.py

```diff
@@ -372,3 +374,20 @@ def get_sagemaker_pipeline(pipeline_name: str):
             f"SageMaker could not find a pipeline with the name {pipeline_name}. Error {error}"
         )
         return None
+
+
+def try_delete_custom_resource(
```

Comment:
`k8s.delete_custom_resource` already has retries, do we need another wrapper method for retry logic?

max_retries = 2 same as changing cfg.DELETE_WAIT_PERIOD to 6 instead of 3? or increasing the period_length

wdyt

Reply:
https://github.com/aws-controllers-k8s/test-infra/blob/0d6bfb1ae7539d9a92dd8a755e6d32ff33b900b0/src/acktest/k8s/resource.py#L196 it only retries on checking status, this will cause it to retry running the deletion command as-well.